assets/elasticsearch/search.py

```
@@ -0,0 +1,47 @@
+            break
+        for result in page["hits"]["hits"]:
+            results.append(result)
+
+
+# Execute the query
+while True:
+    try:
+        execute_query_without_cache(query)
+        print(len(results))
+        break
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)